new-log-viewer/src/contexts/StateContextProvider.tsx

```diff
@@ -275,12 +341,15 @@ const StateContextProvider = ({children}: StateContextProviderProps) => {
             value={{
                 beginLineNumToLogEventNum: beginLineNumToLogEventNumRef.current,
                 fileName: fileName,
+                firstLogEventNumOnPage: firstLogEventNumOnPage.current,
+                lastLogEventNumOnPage: lastLogEventNumOnPage.current,
```

Comment:
Just so we don't forget this discussion: https://github.com/y-scope/yscope-log-viewer/pull/63/files/7bd410b3514b70e3ff86d0cde2cec5e7fa0f8aea#r1768810124

I'm fine if we want to init the refs with literals. e.g.,
```
    const firstLogEventNumOnPage = useRef<number[]>([]);
    const lastLogEventNumOnPage = useRef<number[]>([]);
```